nanoFramework.Azure.Devices.Client/DeviceClient.cs

```diff
@@ -382,25 +387,75 @@ public void RemoveMethodCallback(MethodCallback methodCallback)
         /// Send a message to Azure IoT.
         /// </summary>
         /// <param name="message">The message to send.</param>
+        /// <param name="contentType">Content of the application message.</param>
+        /// <param name="cancellationToken">A cancellation token. If you use the default one, the confirmation of delivery will not be awaited.</param>
+        /// <returns>True for successful message delivery.</returns>
+        public bool SendMessage(
```

Comment:
just to make sure: we are not breaking any compatibility with the changes made here? If yes, then please put back the existing function as well

Reply:
OK I've added it.